### ISSUE TITLE
Automatically maintain indent level

### ISSUE THREAD

**Issue by royappa** [yatki/vscode-surround#64]
If you surround an already indented piece of code, the new lines are added with no indentation.
It would be great if the new lines could be at the current indentation level, and then the surrounded code indented one level inside it. Thanks.

**Comment by yatki** [comment]
duplicate of #8

**Comment by yatki** [comment]
The issue is resolved in version 1.2.0. Thanks for reporting the issue.

The release is available on: 
- [VSCode Marketplace](https://marketplace.visualstudio.com/items?itemName=yatki.vscode-surround)
- [GitHub release](https://github.com/yatki/vscode-surround/releases/tag/v1.2.0)

Cheers 🖖,

**Comment by JordanMandel** [comment]
Is there a setting to turn this behavior off (either globally or per-surround snippet)?  I have a snippet I use all the time which is to add a parenthesis block and it's broken now because it surrounds the whole line even though I've highlighted a specific block of text

If I have this code,
```
if some_condition is not None or some_condition == 42:
   do_something()
```
and then I highlight the code "`some_condition is not None or some_condition == 42`", then it became:
```
if (
  some_condition is not None or some_condition == 42
):
   do_something()
```

Now, no matter what I highlight, it always becomes this, which is not what I want:
```
(
    if some_condition is not None or some_condition == 42
):
   do_something()
```

I probably like this new behavior for *most* snippets, but I would really need to turn it off for others, like the one highlighted above.
